src/Sarif.Viewer.VisualStudio.Core/Sarif/CodeFlow.Extensions.cs

```diff
@@ -39,7 +39,7 @@ public static LocationCollection ToThreadFlowLocationCollection(this CodeFlow co
 
         public static AnalysisStep ToAnalysisStep(this CodeFlow codeFlow, Run run, int resultId, int runIndex)
         {
-            ThreadHelper.ThrowIfNotOnUIThread();
+            // ThreadHelper.ThrowIfNotOnUIThread();
```

Comment:
If not useful, remove it. Also remove the blank line.

Reply:
Why was this removed exactly?

Reply:
this function does not invoke anything related to UI elements, so its not required.